libpod/networking_linux.go

```diff
@@ -547,6 +547,7 @@ func (r *Runtime) setupRootlessPortMappingViaRLK(ctr *Container, netnsPath strin
 		ExitFD:    3,
 		ReadyFD:   4,
 		TmpDir:    ctr.runtime.config.Engine.TmpDir,
+		ChildIP:   "10.0.2.100",
```

Comment:
The more I look at this the less I like it. Would it make sense to add a comment here explaining how/why this magic number was chosen, what its effect is, if other code relies on it (e.g. in rootlesskit)?

Sorry for commenting so late, I was really just hoping someone else would gripe first. Perhaps it just makes intuitive sense to everyone else?

Reply:
thanks for noticing it.  I've added another commit to cleanup these magic IPs from the code